src/libertem/web/server.py

```diff
@@ -37,11 +41,8 @@ def get(self):
         self.render("client/index.html")
 
 
-def make_app(event_registry, shared_state):
-    settings = {
-        "static_path": os.path.join(os.path.dirname(__file__), "client"),
-    }
-    return tornado.web.Application([
+def make_app(event_registry, shared_state, instance_type, instance_config):
+    routes = [
```

Comment:
I added the routes into a list so we can modify and disable routes when in restricted mode.